src/wrapper.test.js

```diff
@@ -255,6 +265,39 @@ describe('Distro initialization', () => {
       });
     });
   });
+
+  describe('On Amazon EKS', () => {
```

Comment:
why do we need double "describe"?

Reply:
in case we'll want to test a non-happy-flow?